test/e2e/support/utils/convert-block.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Convert editor's block type.
+ *
+ * @param {?string} name Block name
```

Comment:
I don't think that block name is optional here.

Reply:
Agree.